### PR TITLE
Adding Inspecting Infrastructure Content

### PR DESCRIPTION
**Pull request by toriancrane**
This PR adds the content for the `Inspecting Infrastructure` topic.

URL Path: `/docs/using-pulumi/inspecting-infrastructure/`

- [x] Awaiting video uploads

### CODE REVIEW

File: content/docs/using-pulumi/inspecting-infrastructure/index.md [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+tags:
+    - resources
+    - infrastructure
+    - fundamentals
+    - tutorial
+---
+
+Pulumi configurations and state data include highly structured information about the resources they manage, such as dependency information, [outputs](/docs/concepts/inputs-outputs/#outputs), and more.
+
+The Pulumi CLI includes some commands for inspecting this data. You can use these to integrate other tools with Pulumi's infrastructure data, or just to gain a deeper or more holistic understanding of your infrastructure.
```
Comment (interurban) [inline content/docs/using-pulumi/inspecting-infrastructure/index.md:20]:
```suggestion
The Pulumi CLI includes commands for inspecting this data. You can use these to integrate other tools with Pulumi's infrastructure data, or just to gain a deeper or more holistic understanding of your infrastructure.
```

File: content/docs/using-pulumi/inspecting-infrastructure/index.md [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+- The [Pulumi CLI](https://www.pulumi.com/docs/install/)
+- An [Amazon Web Services](https://aws.amazon.com/) account
+- The [AWS CLI](https://aws.amazon.com/cli/)
+- Your desired [language runtime installed](/docs/clouds/aws/get-started/begin/#install-language-runtime)
+
+Let's get started!
+
+## Create a New Project
+
+To start, login to the [Pulumi CLI](/docs/cli/commands/pulumi_login/) and ensure it is [configured to use your AWS account](/docs/clouds/aws/get-started/begin/#configure-pulumi-to-access-your-aws-account). Next, [create a new project](/docs/clouds/aws/get-started/create-project/) and replace the default program code with the following:
```
Comment (interurban) [inline content/docs/using-pulumi/inspecting-infrastructure/index.md:41]: Perhaps we explain what this code does ahead of time? and/or also reference that you can run through all these steps on any existing project/stack you already have created?

File: content/docs/using-pulumi/inspecting-infrastructure/index.md (hunk 1 above)
Comment (toriancrane) [reply content/docs/using-pulumi/inspecting-infrastructure/index.md]: updated

File: content/docs/using-pulumi/inspecting-infrastructure/index.md (hunk 2 above)
Comment (toriancrane) [reply content/docs/using-pulumi/inspecting-infrastructure/index.md:47]: updated

**Comment by toriancrane** [comment]
Closing in favor of [branch](https://github.com/pulumi/docs/tree/torian/insp-infra2) with new tutorials experience.

**Pull request closed without merging**